apps/backend/alembic/versions/f4a2b1c8d37e_create_unified_infrastructure_schema.py

```
@@ -0,0 +1,337 @@
+"""Create unified infrastructure schema with TimescaleDB compatibility
+
+Revision ID: f4a2b1c8d37e
+Revises: 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)